docs/source/conf.py

```diff
@@ -41,6 +41,7 @@
               'sphinx.ext.napoleon',
               'sphinx.ext.githubpages',
               'sphinx.ext.autosummary',
+              'sphinx.ext.autosectionlabel'
```

Comment:
This extension is cool, but it leads to a warning: "duplicate label" because of the "Base Wrapper Class"-Headers. I think we should solve that.

Reply:
Fixed in 5de6de5